sway_libs/src/fixed_point/ufp/ufp64/ufp64.sw

```diff
@@ -0,0 +1,267 @@
+library ufp64;
+//! A wrapper around u64 type for a library for Sway for mathematical functions operating with signed 32.32-bit fixed point numbers.
+use core::num::*;
+use std::assert::assert;
+use std::math::*;
+use std::revert::revert;
+use std::u128::U128;
+use std::math::Exponentiate;
+
+pub struct UFP64 {
+    value: u64,
+}
+
+impl UFP64 {
```

Comment:
Can you reorder these functions in order to follow the [contributing book guidelines](https://fuellabs.github.io/sway-applications/book/index.html)?

Reply:
Tried my best to reorder in https://github.com/FuelLabs/sway-libs/pull/31/commits/4835449a514446eeb31959d7c76b16d25dc397e6 , but unfortunately due to known ordering issue it's not always possible